packages/opentelemetry-exporter/package.json

```diff
@@ -0,0 +1,53 @@
+{
+  "name": "opentelemetry-exporter",
+  "version": "1.132.2",
+  "description": "OpenTelemetry Instana Exporter",
+  "keywords": [
+    "opentelemetry",
+    "exporter",
+    "nodejs",
+    "tracing"
+  ],
+  "author": "Willian Carvalho <willian.carvalho@instana.com>",
+  "homepage": "https://github.com/instana/nodejs/blob/main/packages/opentelemetry-exporter/README.md",
+  "license": "MIT",
+  "main": "lib/opentelemetry-exporter.js",
+  "directories": {
+    "lib": "src",
+    "test": "test"
+  },
+  "files": [
+    "src"
```

Comment:
How do we make sure the `dist` folder is included in the released package? Should we add the `dist` folder to the `files` array?

Hint/Note: We could and should test the packaging aspects by running `npm pack` locally and inspecting the resulting tar.gz file plus running `npm install` with that tar.gz file in a temporary directory before actually publishing to the npm registry.

Reply:
> How do we make sure the dist folder is included in the released package

`"prepublishOnly": "npm run build"`

> Hint/Note: We could and should test the packaging aspects by running npm pack locally and inspecting the resulting tar.gz file plus running npm install with that tar.gz file in a temporary directory before actually publishing to the npm registry.

Ok, that's a very good point. Thanks for the hint!

Reply:
My point was not about the prepublish script but about which files will be included in the package and I think dist is missing in the `files` array in packge.json. But I get it now, this is all more preliminary.